app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/bitcoind/WalletResult.scala

```diff
@@ -204,7 +204,30 @@ case class UnspentOutput(
     reused: Option[Boolean])
     extends WalletResult
 
-case class AddressInfoResult(
+sealed trait AddressInfoResult extends WalletResult {
+  def address: BitcoinAddress
+  def scriptPubKey: ScriptPubKey
+  def ismine: Boolean
+  def iswatchonly: Boolean
+  def isscript: Boolean
+  def iswitness: Boolean
+  def iscompressed: Option[Boolean]
+  def witness_version: Option[WitnessVersion]
+  def witness_program: Option[String] // todo what's the correct type here?
```

Comment:
Looks like it is a hash of the key or script

https://github.com/bitcoin/bitcoin/blob/master/src/rpc/util.cpp#L244